broker/applications/osb-broker/src/api-controllers/ServiceBrokerApiController.js

```diff
@@ -414,12 +414,75 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(NotFound, notFound);
   }
 
+  getLastBindingOperation(req, res) {
+    const encodedOp = _.get(req, 'query.operation', undefined);
+    const operation = encodedOp === undefined ? {} : decodeBase64(encodedOp);
+    const guid = req.params.binding_id;
+    const namespaceId = eventmesh.apiServerClient.getNamespaceId(req.params.instance_id);
+    let action, instanceType;
+
+    function done(result) {
+      const body = _.pick(result, 'state', 'description');
+      if (body.state === CONST.APISERVER.RESOURCE_STATE.IN_PROGRESS ||
+        body.state === CONST.APISERVER.RESOURCE_STATE.IN_QUEUE) {
+        body.state = CONST.OPERATION.IN_PROGRESS;
+      }
+
+      logger.debug('RequestIdentity:', _.get(req.headers, CONST.SF_BROKER_API_HEADERS.REQUEST_IDENTITY, 'Absent'), ',returning ..', body);
+      return Promise.try(() => {
+        if (_.get(operation, 'type') === 'delete' && body.state === CONST.OPERATION.SUCCEEDED && resourceGroup === CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR) {
+          logger.debug(`Remove broker finalizer from ${resourceId}`);
+          return this.removeFinalizersFromOSBResource(
+            resourceType,
+            resourceId,
+            namespaceId,
+            _.get(req.headers, CONST.SF_BROKER_API_HEADERS.REQUEST_IDENTITY, 'Absent')
+          );
+        }
+      })
+        .then(() => res.status(CONST.HTTP_STATUS_CODE.OK).send(body));
+    }
+
+    function failed(err) {
+      res.status(CONST.HTTP_STATUS_CODE.OK).send({
+        state: CONST.OPERATION.FAILED,
+        description: `${action} ${instanceType} '${guid}' failed because "${err.message}"`
+      });
+    }
+
+    function gone() {
+      res.status(CONST.HTTP_STATUS_CODE.GONE).send({});
+    }
+
+    function notFound(err) {
+      if (_.get(operation, 'type') === 'delete') {
+        return gone();
+      }
+      failed(err);
+    }
+    const resourceGroup = operation.serviceflow_id ? CONST.APISERVER.RESOURCE_GROUPS.SERVICE_FLOW : CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR;
```

Comment:
Is this check needed?

Reply:
Since we don't use serviceflow any more, I will remove the check and change it to,
```suggestion
    const resourceGroup = CONST.APISERVER.RESOURCE_GROUPS.INTEROPERATOR;
```